doc/reference/core-builtin.md

```diff
@@ -959,7 +959,14 @@ Copies *hash* into a new hash table
   more := list of hash tables
 ```
 
-Creates a new hash table, merging *more* hash tables into *hash*.
+Creates a new hash table, merging *more* hash tables into *hash*. Entries in *hash* take precedence over entries in *more*.
```

Comment:
When there are more than one "more" hash-tables, is the leftmost one preferred? If so, it's not just *hash* taking precedence.

Reply:
The leftmost one is preferred. What's the best way to describe that? And should we try to describe that uniformly we don't have to describe `hash` and `more` separately?

Reply:
Does "Entries in hash tables on the left take precedence over entries on the right." cover this better?